test.sh.in

```diff
@@ -249,32 +272,35 @@ waitForAPI() {
     exit 1
   fi
 }
+if [ "$SKIP_PREP_DEP" == false ] ; then
```

Comment:
I thinking can we add an ”ENV VAR validation” at the beginning of test.sh ?

for example
 if $SKIP_PREP_DEP is NOT defined, and there's no evident that prepareDeps has been run ( example ```build-deps``` folder not exist) , it should be treated as an error (invalid parameter).
moreover,  if $SKIP_PREP_DEP is true and $RUN_CIT_TEST/$RUN_FIT_TEST are both false,  the script will do nothing, there should be at least a warning.

and at the beginning of this script, some necessary ENV VAR like ```REPO_NAME```  & ```WORKSPACE``` should be defined.  some other ENV VAR are optional. but this can be another PR.